rocAL_pybind/amd/rocal/fn.py

```diff
@@ -1139,3 +1139,31 @@ def normalize(*inputs, axes = [], axis_names = "", batch = False, bytes_per_samp
                      "scale": scale , "shift": shift, "ddof": ddof , "epsilon": epsilon, "rocal_tensor_output_type": rocal_tensor_output_type}
     normalize_output = b.audioNormalize(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     return normalize_output
+
+def resample(*inputs, resample_rate=None, rocal_tensor_output_type=types.FLOAT, resample_hint=-1):
+    """
+    Resamples an audio signal.
+
+    The resampling is achieved by applying a sinc filter with Hann window with an extent controlled by the quality argument.
+    """
+    kwargs_pybind = {"input_image0": inputs[0], "resample_rate": resample_rate, "rocal_tensor_output_type": rocal_tensor_output_type, "is_output": False, "resample_hint":resample_hint}
```

Comment:
Please check the params for resample here also and add the missing ones
https://docs.nvidia.com/deeplearning/dali/user-guide/docs/operations/nvidia.dali.fn.audio_resample.html